### DISCUSSION TITLE
The reason for the slowdown in emacs master since January

### DISCUSSION THREAD

**Discussion by paaguti** [minad/jinx#165] [Show and tell]
Emacs with jinx slowed down a lot around January and I had to go back to `flyspell`. After some poking and bugging around, I think I understand the reason.

It is related with an ispell completion-at-point function being introduced in Emacs master. I'm using `company` and `eglot` profusely, and `jinx` is slowed down by this c-a-p-f. 

Just for the record:

```
(setopt text-mode-ispell-word-completion nil)
```

in my `early-init.el` seems to "fix" my setup.

Hope it helps

**Comment by minad** [comment]
Thanks, but I think this is only lightly related to Jinx. This slow Ispell Capf will rather lead to a general slow down if used in conjuction with auto completion ala Company or Corfu. This means **everything** will get slow, typing latency will go up, and not only Jinx will be affected. Such general slow down of Emacs can always be discovered via `M-x profiler-start`. I wonder how you found it? Via the profiler?
